Doc/library/asyncio-dev.rst

```diff
@@ -183,9 +185,10 @@ Detect coroutine objects never scheduled
 ----------------------------------------
 
 When a coroutine function is called and its result is not passed to
-:func:`ensure_future` or to the :meth:`loop.create_task` method,
-the execution of the coroutine object will never be scheduled which is
-probably a bug.  :ref:`Enable the debug mode of asyncio <asyncio-debug-mode>`
+:func:`ensure_future`, :meth:`asyncio.create_task`, or to the 
```

Comment:
`ensure_future()` is something that we need to deprioritize in the docs -- it's low-level and very confusing.  `asyncio.create_task()` is the preferred way to spawn asyncio tasks.  

Same for `loop.create_task()` -- it's a low-level advanced API that few need to ever use directly.